deadline-react/js/ChallengeDetails.js

```diff
@@ -86,46 +85,23 @@ class ChallengeDetails extends React.Component {
   }
 
   render () {
-    const styles = {
-      headline: {
-        fontSize: 24,
-        paddingTop: 16,
-        marginBottom: 12,
-        fontWeight: 400
-      },
-      slide: {
-        padding: 10
-      }
-    }
+    const TabPane = Tabs.TabPane
```

Comment:
Is there any reason to have this in the render() function and not at the top of the file?